### ISSUE TITLE
Fix versions of dependencies 

### ISSUE THREAD

**Issue by kudkudak** [gmum/gmum.r#517]
(and check that CRAN can resolve that successfuly)

**Comment by ktalik** [comment]
TL;DR: Wontfix

From [R-exts manual](https://cran.r-project.org/doc/manuals/R-exts.html#Package-Dependencies):
> The comment should contain a comparison operator, whitespace and a valid version number, e.g. `MASS (>= 3.1-20)`.

There is no information whether dependency version standard is using [semantic versioning](http://semver.org/). I haven't found any information about it anywhere (by greping R source code either). We can ask somewhere but IMHO if it would be present, then there for sure would be an information about it in proper docs.

We can assume that there is no semver. For this reason, we are not able to fix versions on selected release level (see: patch, minor, major in semver). Hence I think we should not fix dependency versions. Such approach could lead to not receiving performance updates. Of course, decision is up to you. Maybe it is better to fix versions with `==` and dont worry about compatibility changes?

Additionally, after taking a look at several largest packages, noone is even doing that + fixing a version from CRAN archive does not work...

`DESCRIPTION`:
```yml
    MASS (== 7.3.33),
```

(7.3.33 is available on [CRAN archive](https://cran.r-project.org/src/contrib/Archive/MASS/))

`devtools::install()`:
`Error : pakiet ‘MASS’ 7.3.40 został znaleziony, ale pakiet == 7.3.33 jest wymagany przez ‘gmum.r’`

Maybe there is a way to do that but I cannot find it. Tried with `Additional_repositories` in `DESCRIPTION` but it appears that [this is not being parsed automatically](http://stackoverflow.com/questions/29419776) which means sadness.
